planning/backlog.md

```diff
@@ -8,42 +8,38 @@ A collection of user stories you can choose from. Stories inside of each priorit
 
 are necessary for basic usability:
 
-- **user story name**: As a [type of user] I want to [do something] so that [I achieve some goal]
-  - Given [context] when [a specific action is performed] then [a set of consequences should occur]
-  - ...
-- **user story name**: As a [type of user] I want to [do something] so that [I achieve some goal]
-  - Given [context] when [a specific action is performed] then [a set of consequences should occur]
-  - ...
-- ...
+- **Website Visitor**: As a Website Visitor I want to have existing header and footer navs so that I have clear navigation around the website:    
+  - [ ] Given when the user visits the website then it must have the main navigation menu displayed in the header and the secondary navigation menu in the footer
+- **Website Visitor**: As a Website Visitor I want the correct links exist in the navs
+  - [ ]  Given when I see the header and the footer, then links to [... specific sections ...] of the website are provided
+- **Website Visitor**: As a Website Visitor I want to know whether I can use the solution on my computer 
+  - [ ] Given when I visit the website then I can see the list of languages supported, and the technical requirements
+- **Website Visitor**: As a Website Visitor I want to know how to install Loruki on my computer so that I could use it   
+  - [ ] Given when the user visits the home page then he can see brief information ont how to instal Loruki
+  - [ ] Given when the user visits the [other] pages then he can see all the necessary instructions. 
```

Comment:
Given when the user visits the Docs page then he can see all the necessary instructions.